src/main/java/teammates/logic/core/FeedbackQuestionsLogic.java

```diff
@@ -122,6 +122,31 @@ public FeedbackQuestionAttributes getFeedbackQuestion(
         return questions;
     }
 
+    /**
+     * Gets the number of questions for student to answer from a map of giver type to question counts.
+     */
+    public long getFeedbackQuestionForStudentCountFrom(Map<FeedbackParticipantType, Integer> giverTypeCounts) {
+        return giverTypeCounts.getOrDefault(FeedbackParticipantType.STUDENTS, 0)
```

Comment:
I think the return type should be `int` since the value of the map is an `Integer`